MANIFEST.in

```diff
@@ -3,6 +3,7 @@ prune docs/_build
 graft zodburi
 
 include README.rst
+include RELEASING.txt
```

Comment:
This should not be included in the manifest because it is intended only for maintainers and not for release to the general public. I would exclude it.

Reply:
The reason this file is included in MANIFEST.in is because otherwise check-manifest complains about it being missing here:

```
(r.py3) kirr@deco:~/src/wendelin/z/zodburi$ check-manifest 
lists of files in version control and sdist do not match!
missing from sdist:
  RELEASING.txt
suggested MANIFEST.in rules:
  include *.txt
```

I think it is ok to include RELEASING.txt into published tarball.

Reply:
I assume that this would work.

```
exclude RELEASING.txt
```

Reply:
indeed, thanks.